samples/grpc/local-drone-control-scala/src/main/protobuf/common/coordinates.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "common.proto";
+
+package common;
+
+// generic messages, shared between local-drone-control and restaurant-drone-deliveries
```

Comment:
The ownership/duplication of proto message definitions felt a bit messy, I think we need a recommendation on how to deal with those (prepackaged, shared directory, something less copy-paste-by-hand)

Reply:
add to the doc ticket todo